crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -287,33 +291,32 @@ fn authenticate_incoming_app_connection<A: InterfaceApi<Auth = AppAuthentication
     task_list_lock.push(join_handle);
 }
 
-/// Polls for messages coming in from the external client while simultaneously
-/// polling for signals being broadcast from the Cells associated with this
-/// App interface.
-fn spawn_recv_incoming_msgs_and_outgoing_signals<A: InterfaceApi>(
```

Comment:
Very messy diff here, but this function has been split into two because it did two jobs